sessions/base.py

```
@@ -0,0 +1,130 @@
+            finally:
+                self.widgets.remove(widget)
+                logger.info(f"Removed widget {widget.__class__.__name__} from session")
+
+    async def _notify_widgets_conversation_item_added(
+        self, event: ConversationItemAddedEvent
+    ) -> None:
+        """Notify all widgets about conversation item added."""
+        for widget in self.widgets:
+            try:
+                await widget.on_conversation_item_added(event)
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)